crates/biome_formatter_test/src/prettier/prepare_tests.js

```diff
@@ -62,31 +62,52 @@ async function traverseDir(dir, input_config) {
 			const snapshot = require(snapshotPath);
 
 			const key = `${file} format 1`;
-			let snapshotContent = snapshot[key];
 
-			if (snapshotContent !== undefined) {
+			if (key in snapshot) {
+				let snapshotContent = String(snapshot[key]);
```

Comment:
Checked `key` using `in` operator instead of `snapshotContent !== undefined` since `snapshotContent` is only used within the if statement so narrowing the scope. 
Also, used `String(snapshot[key])` to have editor support. We do know `snapshot[key]` is `string`, but editor wouldn't know the type until we kindly let editor like this due to JS file (editor thinks it's `any`).